srcpkgs/gpodder/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'gpodder'
 pkgname=gpodder
-version=3.10.17
+version=3.10.21
 revision=1
 wrksrc="gpodder-${version}"
-hostmakedepends="python3 intltool"
+hostmakedepends="python3 intltool which"
 depends="eyeD3 gtk+3 hicolor-icon-theme python3-dbus python3-gobject
  python3-html5lib python3-mygpoclient python3-podcastparser"
-checkdepends="${depends} python3-MiniMock python3-coverage desktop-file-utils"
+checkdepends="${depends} python3-MiniMock python3-coverage desktop-file-utils
+ python3-pytest python3-pytest-httpserver python3-pytest-cov python3-requests"
 short_desc="Podcast client"
 maintainer="bra1nwave <bra1nwave@protonmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/gpodder/gpodder"
 distfiles="${homepage}/archive/${version}.tar.gz"
```

Comment:
Can we replace `${homepage}` with `https://github.com/gpodder/gpodder`?

Reply:
Ok :)